spring-cloud-dataflow-server/docker-compose.yml

```diff
@@ -1,6 +1,15 @@
 version: '3'
 
 services:
+  mysql:
+    image: mysql:5.7.25
+    environment:
+      MYSQL_DATABASE: dataflow
+      MYSQL_USER: root
+      MYSQL_ROOT_PASSWORD: rootpw
```

Comment:
Not sure what the best practice is when using user/pass in Docker Compose experience. Folks question the reasoning behind having this as clear-text. We encode it in K8s world; I wonder if there's anything similar in plain Docker world.

Reply:
agreed plaintext is not a production grade thing, but for a "quick start" it requires no work on the end user side.

docker secrets could be an option (https://docs.docker.com/compose/compose-file/#secrets) - (https://hub.docker.com/_/mysql) but would add extra steps to explain and have the end user setup - not sure it would make for a good "quick start" experience

if its commonly asked how to avoid plain text, this might be a candidate for an entry in the customization section